playbook-website/app/views/pages/visual_guidelines_partials/_pb_doc_shadow.html.erb

```diff
@@ -0,0 +1,82 @@
+<%= pb_rails("title", props: {margin_top: "xl", text: "Shadows", tag: "h1", size: 1 }) %>
+
+<%= pb_rails("body", props: { margin_y: "md"} ) do %>
+  Shadows are a night way of showing depth. Modals, buttons, & popovers are examples of items that need to have a shadow to indicate their position relative to others. Our tokens help you expand those to the other custom elements you build using playbook.
+<% end %>
+
+<br>
+
+<h4 aria-label="online" class="pb_title_kit_4 pt_sm">Tokens</h4>
```

Comment:
Wondering why is this not using our `Title` kit?

Reply:
Oops sorry! I'll switch that to the Title kit. Thank you!

Reply:
Fixed!  Replaced the h4 with the Title kit!